configs/katello/4.1.yaml

```diff
@@ -0,0 +1,113 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.1.0:
+    :redmine_version_id: 1351
+:prior_releases:
+  :4.0.0:
+    :redmine_version_id: 1177
+  :3.17.3:
+    :redmine_version_id: 1365
+  :3.18.2:
+    :redmine_version_id: 1353
+  :3.18.2.1:
+    :redmine_version_id: 1384
+  :3.18.3:
+    :redmine_version_id: 1370
+:code_name:
+:mash_scripts:
+  - katello-4.1-rhel7
+  - katello-4.1-thirdparty-candlepin-rhel7
+  - katello-4.1-el8
+  - katello-candlepin-4.1-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.1
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.5
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.0.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.4
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.7.3
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
```

Comment:
not sure which if any of these versions need to change

Reply:
Best to check with the maintainers. Some of all of those release probably have new releases available or need to be created for 4.1

I can say that there will be new versions of hammer-cli-katello and foreman_virt_who_configure for sure because we've made changes there recently